content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -48,6 +48,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
   is an ingress controller driving [Kong Gateway](https://konghq.com/kong/).
 * The [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx-ingress-controller/)
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
+* The [Pomerium Ingress Controller](https://www.pomerium.com/docs/k8s/ingress.html) provides ingresses that work with the [Pomerium](https://pomerium.com/) context-aware proxy.
```

Comment:
Could you reword “provides ingresses” @alexfornuto? How about “lets you create Ingresses”?

(ideally, it would be “lets you define IngressClasses” but I from the docs I skimmed, I'm not sure that Pomerium ingress controller supports IngressClass properly)

Reply:
Sorry for the delayed response. How about:

```suggestion
* The [Pomerium Ingress Controller](https://www.pomerium.com/docs/k8s/ingress.html) is based on [Pomerium](https://pomerium.com/), which offers context-aware access policy.
```

We do support IngressClass, though, so if you see a shortcoming in our implementation we'd be interested in addressing it.

Reply:
Revised suggestion also sounds OK.

BTW, https://www.pomerium.com/docs/k8s/ingress.html doesn't mention any IngressClass at all and (as part of not mentioning it) doesn't let a user know what the `.spec.controller` should be for any IngressClass they define.

Reply:
I think the specific definition you're looking for is set in our [helm chart](https://github.com/pomerium/pomerium-helm/blob/master/charts/pomerium/templates/ingressclass.yaml). You can see an example of the `spec.controller` defined as `pomerium.io/ingress-controller` in the [controller's README](https://github.com/pomerium/ingress-controller#ingressclass).